Task/Task-TestOrderFormCompleted-StorageOfMaterial-Example.json

```diff
@@ -0,0 +1,57 @@
+{
+    "resourceType": "Task",
+    "id": "Task-TestOrderFormCompleted-StorageOfMaterial-Example",
+    "basedOn": [
+        {
+            "reference": "ServiceRequest/ServiceRequest-TestOrderForm-StorageOfMaterial-Example"
+        }
+    ],
+    "status": "completed",
+    "businessStatus": {
+      "coding": [
+        {
+          "system": "https://fhir.nhs.uk/CodeSystem/business-status-genomics",
+          "code": "SpecimenReceived",
+          "display": "Specimen Received"
+        }
+      ]
+    },
+    "intent": "order",
+    "priority": "routine",
+    "code": {
```

Comment:
I'm not sure any sample prep was done, only storage so I would change the code to Request/sample alignment?

Reply:
Changed to Request & Sample Alignment. However, does that not suggest we are asking for further info? 
My earlier assumption was sample storage should be part of preparation, while sample processing would mean work has started on it.

Reply:
Not sure to be honest, sample preparation may include splitting samples into multiple specimens, fixing, centrifuging etc. which I'm not sure are done just on just sample storage, so best to keep it at an earlier stage